.eslintrc.js

```diff
@@ -3,4 +3,9 @@ const { config } = require('@dhis2/cli-style')
 module.exports = {
     parser: 'babel-eslint',
     extends: ['eslint:recommended', 'plugin:react/recommended', config.eslint],
+    settings: {
+        react: {
+            version: 'detect',
```

Comment:
This could be 16.3 for now.

Reply:
👍 https://github.com/dhis2/ui-core/pull/443